### DISCUSSION TITLE
Spectroscopic data as input

### DISCUSSION THREAD

**Discussion by shivathanay-p** [mboquien/cigale#100] [Q&A] [answered]
Instead of providing fluxes in different bands, is it possible to give the whole spectrum of the galaxy as an input?

**Comment by mboquien** [comment, marked as answer]
You can do something like that defining very narrow filters and adding them to the database.

**Reply by Penguin2542** [reply]
Could you please elaborate a bit on how to do it?

**Reply by mboquien** [reply]
You can create filters following the format of the files in `database_builder/`. Then you can add filters with `pcigale-filters-add FILE1 FILE2 …`. Then you simply need to resample the spectrum on these filters and use these fluxes as regular photometric observations in the input flux file.

**Reply by shivathanay-p** [reply]
I want to use CIGALE on the SDSS data. I was hoping there was a way to give the complete spectra as input, but if I were to create many narrow filters instead, how do I define the second column in the filter file, which I assume is for the transmittance values for each wavelength?

**Reply by mboquien** [reply]
For instance a filter with a width of 5 Å between 6000 and 6005 Å would be :

```
5999.99 0.0
6000.00 1.0
6005.00 1.0
6005.01 0.0
```

**Reply by shivathanay-p** [reply]
As with the other wide-band filters in `database_builder/`, are the transmittance values obtained through some calibration?

**Reply by mboquien** [reply]
These are relative transmissions. They are renormalized when imported to produce fluxes in the correct units while minimizing the number of operations.
